src/meta/meta_backup_service.h

```diff
@@ -79,15 +79,10 @@ struct backup_info
         backup_id, start_time_ms, end_time_ms, app_ids, app_names, info_status)
 };
 
-// Attention: backup_start_time == 24:00 is represent no limit for start_time, 24:00 is mainly saved
-// for testing
-//
-// current, we don't support accurating to minute, only support accurating to hour, so
-// we just set minute to 0
 struct backup_start_time
 {
-    int32_t hour;   // [0 ~24)
-    int32_t minute; // [0 ~ 60)
+    int32_t hour;
```

Comment:
It's better to change the type of hour and minute to `uint8_t`

Reply:
`dsn::utils::time_ms_to_date_time` use `int32_t` for hour, min, sec, so we also use `int32_t` here.
Maybe we should consider use `uint8_t` to replace `int32_t` in `dsn::utils::time_ms_to_date_time`, if not , even if we changed  interger type here, type conversions would still be needed.